keyboards/mschwingen/modelm/keymaps/default/keymap.c

```diff
@@ -0,0 +1,92 @@
+    ______,  DYN_MACRO_PLAY1, DYN_MACRO_PLAY2, DYN_REC_START1, DYN_REC_START2, DYN_REC_STOP, ______, ______ , ______ , ______, ______, ______, ______, ______, ______, ______,
+    KC_LOCK, ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______, ______ ,  ______ , ______ , ______ , ______ ,
+    ______,  ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______, ______ ,  ______ , ______ , ______ , ______ ,
+    KC_CAPS, ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ ,                            ______ , ______ , ______ ,
+    ______,  ______, ______, ______, ______, ______ , ______ , ______ , TG(_MS), ______ , ______ , ______ ,           ______ ,                    KC_AUDIO_VOL_UP,       ______ , ______ , ______ , ______ ,
+    ______,          ______,                          KC_MEDIA_PLAY_PAUSE ,                        ______ ,          ______ , KC_MEDIA_PREV_TRACK , KC_AUDIO_VOL_DOWN , KC_MEDIA_NEXT_TRACK , ______ ,          ______
+  ),
+
+  [_MS] = LAYOUT( /* mouse key layer */
+  // Cursor: mouse, INS/HOME/PgUp: Mouse Accel, Del, End, PageDn: mouse buttons
+    TG(_MS),  ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ ,
+    ______,  ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , KC_MS_ACCEL0, KC_MS_ACCEL1, KC_MS_ACCEL2, ______ , ______ , ______ , ______ ,
+    ______,  ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , KC_MS_BTN1,   KC_MS_BTN3,   KC_MS_BTN2,   ______ , ______ , ______ , ______ ,
+    ______,  ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ ,                                           ______ , ______ , ______ ,
+    ______, ______ , ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ ,          ______ ,               KC_MS_UP,                   ______ , ______ , ______ , ______ ,
+    ______,          ______,                          ______ ,                                     ______ ,          ______ , KC_MS_LEFT,   KC_MS_DOWN,   KC_MS_RIGHT,  ______ ,          ______
+  ),
+
+};
+
+
+void keyboard_post_init_user(void) {
+  // Customise these values to desired behaviour
+  //debug_enable=true;
+  //debug_matrix=true;
+  //debug_keyboard=true;
+  //debug_mouse=true;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  // If console is enabled, it will print the matrix position and status of each key pressed
+#ifdef CONSOLE_ENABLE
+    dprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+#endif
```

Comment:
If you're using dprintf, you shouldn't need the console enable parts, actually.

Reply:
OK. I also changed other occurrences from uprintf to dprintf to fix compilation errors when CONSOLE_ENABLE is off.